inc/thorvg.h

```diff
@@ -1341,6 +1341,53 @@ class TVG_API Picture final : public Paint
      * @return The type id of the Picture class.
      */
     static uint32_t identifier() noexcept;
+    
+    
+    /**
+     * @brief Specify segment by marker
+     *
+     * @param[out] marker segment name
+     *
+     * @note Set empty to reset specified segment
+     *
+     * @note Experimental API
+     */
+    Result segment(const std::string& marker);
```

Comment:
1. This segment API should be moved to the class LottieAnimation in thorvg_lottie.h
2. thorvg would not use string, please replace the string with const char*